src/raft.h

```diff
@@ -328,6 +329,25 @@ struct raft_append_entries_result
 };
 #define RAFT_APPEND_ENTRIES_RESULT_VERSION 1
 
+typedef unsigned int checksum_t;
+typedef unsigned long int pageno_t;
```

Comment:
I would prefer fixed-size integers here, also `pageno_t` should be only 32 bits (that this is enough is fixed by the SQLite file formats).